File: tests/fixtures/swift/production_api_service.swift
class AppConfig {
    var host: String = ""
    var port: Int = 0
    var databaseUrl: String = ""
    var redisUrl: String = ""
    var jwtSecret: String = ""
    var logLevel: String = ""
    var maxConnections: Int = 0
    var timeoutSecs: Int = 0
    var retryCount: Int = 0
    var cacheTtl: Int = 0
    var rateLimit: Int = 0
    var uploadDir: String = ""
    var corsOrigin: String = ""
}

func dispatchCommand(cmd: String) -> String {
    switch cmd {
    case "start":
        return "starting"
    case "stop":
        return "stopping"
    case "restart":
        return "restarting"
    case "status":
        return "checking"
    case "deploy":
        return "deploying"
    case "rollback":
        return "rolling back"
    default:
        return "unknown"
    }
}

func processEvent(data: [Int]) -> Int {
    let a = data[0]
    let b = data[1]
    let c = data[2]
    let d = data[3]
    let e = data[4]
    let f = data[5]
    let g = data[6]
    let h = data[7]
    if a > 100 {
        return -1
    }
    if b > 200 {
        return -2
    }
    let result = a + b + c + d
    let extra = e + f + g + h
    return result + extra
}

File: tests/fixtures/ruby/production_api_service.rb
class AppConfig
  attr_accessor :host, :port, :database_url, :redis_url, :jwt_secret,
                :log_level, :max_connections, :timeout_secs, :retry_count,
                :cache_ttl, :rate_limit, :upload_dir, :cors_origin
end

def dispatch_command(cmd)
  case cmd
  when "start" then "starting"
  when "stop" then "stopping"
  when "restart" then "restarting"
  when "status" then "checking"
  when "deploy" then "deploying"
  when "rollback" then "rolling back"
  end
end

def process_event(data)
  a = data[0]
  b = data[1]
  c = data[2]
  d = data[3]
  e = data[4]
  f = data[5]
  g = data[6]
  h = data[7]
  return -1 if a > 100
  return -2 if b > 200
  result = a + b + c + d
  extra = e + f + g + h
  result + extra
end
